apps/ftu/js/ui.js

```diff
@@ -431,6 +431,9 @@ var UIManager = {
       case 'metrics-basic':
       case 'metrics-enhanced':
       case 'metrics-none':
+        this.updateSetting('hud.hide', (event.target.value === 'Enhanced'));
+        this.updateSetting('devtools.overlay',
```

Comment:
I think that makes sense, but I'm not sure it really matters as it's FTU.  But I can add that as a safeguard.  I think the only time it would matter if if we turned on HUD via the Dogfood flag.